autodispose/build.gradle

```diff
@@ -26,13 +27,19 @@ test {
   testLogging.showStandardStreams = true
 }
 
+lintOptions {
+  abortOnError true
+  warningsAsErrors true
+}
+
 dependencies {
   annotationProcessor deps.build.nullAway
   testAnnotationProcessor deps.build.nullAway
 
   api deps.rx.java
 
   errorprone deps.build.errorProne
+  lintChecks project(':static-analysis:autodispose-lint')
```

Comment:
I haven't gotten this to **actually** report anything though - https://issuetracker.google.com/issues/112526243

Reply:
I thought this was just for projects consuming the lints? Not for packaging

Reply:
It's also for packaging (prior art in [Butterknife](https://github.com/JakeWharton/butterknife/blob/master/butterknife-runtime/build.gradle#L40)) in a library.